src/components/TODOList/ListView.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import TodoItem from './TODOItem';
+
+const ListView = (props) => {
+
+    let sortedTodos = [...props.visibleTODOs];
```

Comment:
you do not need to use the spread operator here to create a new array sortedTodos, just use `props.visibleTODOs` where needed

Reply:
Ok. Advice taken. Thanks!